builtin/v14/miner/policy.go

```diff
@@ -107,6 +107,16 @@ var MaxPreCommitRandomnessLookback = builtin.EpochsInDay + ChainFinality // PARA
 // (2) prevents a miner attempting a long fork in the past to insert a pre-commitment after seeing the challenge.
 var PreCommitChallengeDelay = abi.ChainEpoch(150) // PARAM_SPEC
 
+// The maximum number of sector commitments in a single batch for calling ProveCommitSectorsNI.
+// Same as PreCommitSectorBatchMaxSize for consistency.
+var ProveCommitNiSectorBatchMaxSize = 256 // PARAM_SPEC
+
+// Maximum number of epochs within which to fetch a valid seal randomness from the chain for
+// a non-interactive PoRep proof. This balances the need to tie the seal to a particular chain with
+// but makes allowance for service providers to offer pre-sealed sectors within a larger window of
+// time.
+var MaxProveCommitNiLookback = abi.ChainEpoch(30*builtin.EpochsInDay) + ChainFinality // PARAM_SPEC
```

Comment:
this will likely become 180 after a fip change

Reply:
```suggestion
var MaxProveCommitNiLookback = abi.ChainEpoch(180*builtin.EpochsInDay) + ChainFinality // PARAM_SPEC
```

Reply:
need to remove finality too

Reply:
Ah, correct! Updated the suggested change.

Reply:
done